### PR TITLE
Project: Tic-Tac-Toe game

### PR DESCRIPTION
**Pull request by hemantso**
## Project: Tic-Tac-Toe game
- Readme file with game instruction
-  Class Game and different method used defined

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,238 @@
...
+```
+
+
+#### `bin/main`
+
+This is our main executable and will be how we run our game.
+
+#### `bin/main.rb`
+
+Our main `Game` class will be defined here with all the data and logic required to play a game of tic tac toe via instances of `Game`.
```
Comment (mosesogwo) [inline README.md]: Are these referring to two different files?

File: README.md [hunk 2]
```diff
@@ -1 +1,238 @@
...
+We'll be getting user input via `gets` and a player will choose a position by entering 1-9. Our program will then fill out the appropriate position on the board with the player's token.
+
+We will keep track of which player's turn it is and how many turns have been played. We will check to see, at every turn, if there is a winner. If there is a winner, we'll congratulate them. If there is a tie, we will inform our players.
+
+## Instructions
+
+### `Game` class.
+
+Open up `./bin/main.rb`. You'll be defining the main game class, `Game` in `bin/main.rb`. Without that file defining a `Game` class, everything will break.
+
```
Comment (mosesogwo) [inline README.md]: This information will change since Game class will not be residing in main.rb

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Hi @hemant-soni-vst-au4, please complete the previous milestone first so you can provide correct information in the README :exclamation:**
- See my comments on your README
- Do you really need to explain each method in the README? Your code itself and commit messages should sufficiently describe what is happening in the project.

### Status: Changes requested.

**Comment by uryelah** [comment]
Hello **Hemant Soni**, I'm Sarah, your last reviewer for this PR 👩‍🚀  

Great job with the Readme! It's very succinct.

---------

## **Your project is approved! 🎊 🥅 💛**

**You may close this Pull Request and merge the changes**

---------

**Keep being an awesome dev and good luck with your next project!**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Pull request merged**